lib/Service/SignFileService.php

```diff
@@ -301,7 +301,7 @@ public function validateExistingFile(array $data) {
 			$this->validateHelper->validateLibreSignNodeId($data['file']['fileId']);
 			$this->validateHelper->iRequestedSignThisFile($data['userManager'], $data['file']['fileId']);
 		} else {
-			throw new \Exception($this->l10n->t('Inform or UUID or a File object'));
+			throw new \Exception($this->l10n->t('Inform or UUID or a file object'));
```

Comment:
This `file` is upper case first, is reference to `File` object in request of API. Is a pattern write first letter in uppercase of objects.

```suggestion
			throw new \Exception($this->l10n->t('Inform or UUID or a File object'));
```

Reply:
Done 😉.